client/chatroomwidget.cpp

```diff
@@ -25,6 +25,7 @@
 #include <QtWidgets/QAction>
 #include <QtWidgets/QFileDialog>
 #include <QtWidgets/QApplication>
+#include <QLocale>
```

Comment:
```suggestion
#include <QtCore/QLocale>
```
And even better, move this down to group with other QtCore #includes.